segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/StorageManagerLayoutType.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.segmentstore.storage;
+
+/**
+ * Type of Storage metadata layout to use.
+ */
+public enum StorageManagerLayoutType {
+    /**
+     * Uses RollingStorage based layout.
+     */
+    LEGACY,
```

Comment:
I am not sure this is a good name for the public audience.
What about:
- V07
- RAW
- SIMPLE

Something that makes it clear that is not TABLE_BASED but as it is the default it cannot smell like 'this is old stuff, probably no more very much supported'

Reply:
I added a similar comment in the config file. Let's be consistent. A bit of documentation pointing to the (soon-to-be-defunct) RollingStorage.java would help too.

Reply:
fixed